conf/modules.config

```diff
@@ -475,11 +467,11 @@ process {
     withName: '.*ANNOTATE_SNV:ENSEMBLVEP' {
         ext.prefix = { "${vcf.simpleName}_rohann_vcfanno_filter_vep" }
         ext.args   = [
-            '--dir_plugins vep_cache/Plugins',
-            '--plugin LoFtool,vep_cache/LoFtool_scores.txt',
-            '--plugin pLI,vep_cache/pLI_values_107.txt',
-            '--plugin SpliceAI,snv=vep_cache/spliceai_21_scores_raw_snv_-v1.3-.vcf.gz,indel=vep_cache/spliceai_21_scores_raw_snv_-v1.3-.vcf.gz',
-            '--plugin MaxEntScan,vep_cache/fordownload,SWA,NCSS',
+            '--dir_plugins cache/Plugins',
+            '--plugin LoFtool,cache/Plugins/LoFtool_scores.txt',
+            '--plugin pLI,cache/Plugins/pLI_values_107.txt',
+            '--plugin SpliceAI,snv=cache/Plugins/spliceai_21_scores_raw_snv_-v1.3-.vcf.gz,indel=cache/Plugins/spliceai_21_scores_raw_snv_-v1.3-.vcf.gz',
+            '--plugin MaxEntScan,cache/Plugins/fordownload,SWA,NCSS',
```

Comment:
```suggestion
```
Let's skip these plugins for now as they are for predicting splicing

Reply:
This will also make life easier for us, as some of the plugins weren't inside the vet refs to start with